src/sentry_auth_saml2/forms.py

```diff
@@ -64,7 +74,6 @@ def process_metadata(form_cls, request, helper):
         return form
 
     helper.bind_state('idp', data)
-    helper.bind_state('contact', request.user.email)
```

Comment:
This was also a change made in https://github.com/getsentry/sentry/pull/6206, contact is not needed here.